### ISSUE TITLE
references seem to be broken

### ISSUE THREAD

**Issue by fterpstra** [Geonovum/respec#156]
## Important info

* in https://geonovum.github.io/KP-APIs/#bib-rfc7662 and https://geonovum.github.io/whitepaper-standaarden/
* ReSpec version: 22.4.0
* I tried in chrome 69.0.3497.100 and in Edge 41.16299.248.0

## Description of problem

What happened: I can include references both to localbiblio and to the specref database. Both seem to be generated correctly inline with the text. A link appears. however the link (for instance geonovum.github.io/whitepaper-standaarden/#bib-rfc7159) does not go anywhere. There is no references section at the end of the document to point to. 

But what I was expecting?:
A references section at the end of the document where the links point to.
Optional, steps to reproduce:
 1. look at any references in brackets within the mentioned repo
 2. click on the link and see it going nowhere
 3. look at the end of the document and see no references section

**Comment by thijsbrentjens** [comment]
Also seems to be an issue in version 23.0.4 of Geonovum build (locally tested)

**Comment by marcoscaceres** [comment]
Can you check that the markup is valid... something is really weird there in that References are ending up in a sub section... which, um, should never happen 🤔

**Comment by thijsbrentjens** [comment]
Thanks for the tip. I validated the markup for https://geonovum.github.io/whitepaper-standaarden/ , but only got 2 warnings that there are sections without proper headings. In this doc there are no bibrefs either, so we have an issue here. I understand that you don't have it at W3C.

The doc at: https://geonovum.github.io/KP-APIs/ seems broken indeed, also the index is broken. This doc contains several errors, so a need to fix that first (@fterpstra could you check? I created a snapshot of the doc and validated that with https://validator.w3.org . Some links have spaces in the URL, which is not valid edit: and there is a dublicate id "abstract".)

**Comment by lvdbrink** [comment]
@thijsbrentjens strange...
https://geonovum.github.io/whitepaper-standaarden/ is a stable doc and the References section worked in the past.

**Comment by marcoscaceres** [comment]
Hmm... very odd. @thijsbrentjens, could you try making a very small document as a test case? It's ok on the W3C side, so we need to figure out if something is causing it in one of the documents or if a merge went bad along the way.

**Comment by thijsbrentjens** [comment]
I think I have the solution.

Backgroud: I created a simple test case. When I tested this with the W3c profile, the biblio was shown. I checked the geonovum profile and noticed we missed a reference to a module (core/biblio-render). So I apparently missed this change in our own profile: https://github.com/w3c/respec/commit/0998305e6a463c617471dc01cdcf82abc9d89c33#diff-77169715f1ec50be3e8f45080345053f

Will add that to our checklist when merging new W3c code.

@fterpstra @lvdbrink I need to do some more testing, will create a new build (on test) after that
(edit) @marcoscaceres thanks again for your help
